autosportlabs/racecapture/settings/prefs.py

```
@@ -346,6 +365,19 @@ def set_pref(self, section, option, value):
         if value != current_value:
             self.dispatch('on_pref_change', section, option, value)
 
+    def set_pref_list(self, section, option, value):
+        """
+        Set a preference value by list
+        :param section the configuration section for the preference
+        :type string
+        :param option the option for the section
+        :type string
+        :param value the preference value to set
+        :type value list (list of strings)
+        """
+        self.set_pref(section, option, ','.join(value))
```

I could catch, log and re-throw so crash handler gets it, or, I could catch, log, and swallow the error. 

        try:
            self.set_pref(section, option, ','.join(value))
        except TypeError:
            Logger.error('UserPrefs: failed to set preference list for {}:{} - {}'.format(section, option, value))


Thoughts on one approach vs the other? Saving a preference isn't critical for operation; it's a convenience for the user. If we re-throw it could disrupt the function attempting to run.  if we swallow it, we won't find out about it via CrashHandler.

Have we been using CrashHander in a soft way yet?